frontend/src/component/operator-wizard/element-component/field-factory/DefaultTextField.js

```diff
@@ -1,71 +1,120 @@
+              // onChange={this.onChangeText}
+              onBlur={this.onBlurText}
+              jsonpath={this.props.fieldDef.jsonPath}
+              // value={((this.props.fieldDef.default!==undefined ) ? this.props.fieldDef.default:this.props.fieldDef.value)}
+              defaultValue={this.value}
+              {...this.props.attrs}
+            />
+          </Tooltip>
         </FormGroup>
       );
     } else {
       return (
         <FormGroup
           label={this.props.fieldDef.label}
-          fieldId={this.props.ids.fieldGroupId}
+          id={this.props.ids.fieldGroupId}
           key={this.props.ids.fieldGroupKey}
+          helperTextInvalid={this.errMsg}
+          fieldId={this.props.ids.fieldId}
+          isValid={this.isValid}
         >
-          <TextInput
-            type="text"
-            id={this.props.ids.fieldId}
-            key={this.props.ids.fieldKey}
-            aria-describedby="horizontal-form-name-helper"
-            name={this.props.fieldDef.label}
-            // onChange={this.onChangeText}
-            onBlur={this.onBlurText}
-            jsonpath={this.props.fieldDef.jsonPath}
-            // value={((this.props.fieldDef.default!==undefined ) ? this.props.fieldDef.default:this.props.fieldDef.value)}
-            defaultValue={this.props.fieldDef.value}
-            //value={this.value}
-            {...this.props.attrs}
-          />
+          <Tooltip
+            position="left"
+            content={<div>{this.props.fieldDef.description}</div>}
+            enableFlip={true}
+            style={{
+              display:
+                this.props.fieldDef.description !== undefined &&
+                this.props.fieldDef.description !== ""
+                  ? "block"
+                  : "none"
+            }}
+          >
+            <TextInput
+              type="text"
+              id={this.props.ids.fieldId}
+              key={this.props.ids.fieldKey}
+              aria-describedby="horizontal-form-name-helper"
+              name={this.props.fieldDef.label}
+              // onChange={this.onChangeText}
+              onBlur={this.onBlurText}
+              jsonpath={this.props.fieldDef.jsonPath}
+              // value={((this.props.fieldDef.default!==undefined ) ? this.props.fieldDef.default:this.props.fieldDef.value)}
+              defaultValue={this.value}
+              //value={this.value}
+              {...this.props.attrs}
+            />
+          </Tooltip>
         </FormGroup>
       );
     }
   }
 
   onBlurText = event => {
     let value = event.target.value;
-    if (value !== undefined && value !== null && value !== "") {
+    if (value !== undefined && value !== null) {
+      this.isValidField(value);
       this.props.fieldDef.value = value;
-      // this.props.fieldDef.default="";
-
-      //document.getElementById(event.target.id).value = value;
+      this.value = value;
     }
   };
+
+  isValidField(value) {
```

Comment:
There are quite a few issues with this, but given the current status, we should merge and address them in follow-ups.